### PR TITLE
Feature/tao 8134 integrate the item previewer

### PR DESCRIPTION
**Pull request by Sunbus**
Task: https://oat-sa.atlassian.net/browse/TAO-8134

Related PRs:
- https://github.com/oat-sa/extension-tao-testqti-previewer/pull/21
- https://github.com/oat-sa/extension-tao-item/pull/362

In this PR we are trying to implement new preview item component to items menu in back-office.

### CODE REVIEW

File: views/js/qtiCreator/plugins/menu/preview.js [hunk 1]
```diff
@@ -29,10 +29,10 @@ define([
     'i18n',
```
Comment (ferenckiss89) [inline views/js/qtiCreator/plugins/menu/preview.js]: update the license year pls to 2016-2019

File: views/js/qtiCreator/plugins/menu/preview.js [hunk 2]
```diff
@@ -29,10 +29,10 @@ define([
...
     'core/plugin',
     'ui/hider',
-    'taoItems/preview/preview',
+    'taoItems/previewer/factory',
     'tpl!taoQtiItem/qtiCreator/plugins/button',
     'util/url'
```
Comment (oatymart) [inline views/js/qtiCreator/plugins/menu/preview.js]: `util/url` looks to be a redundant dependency now. You can remove it.

File: views/js/qtiCreator/plugins/menu/preview.js [hunk 3]
```diff
@@ -57,12 +57,12 @@ define([
...
+              	var type = 'qtiItem';
 
-                //TODO move away the URLs !!!
-                preview.init(urlUtil.build(itemCreator.getConfig().properties.previewUrl, {
-                    uri: uri
-                }));
-                preview.show();
+                previewerFactory(type, uri, { }, {
+                  readOnly: true,
+                  fullPage: true
```
Comment (oatymart) [inline views/js/qtiCreator/plugins/menu/preview.js]: Indentation not quite right?

File: views/js/qtiCreator/plugins/menu/preview.js (hunk 1 above)
Comment (Sunbus) [reply views/js/qtiCreator/plugins/menu/preview.js]: fixed

File: views/js/qtiCreator/plugins/menu/preview.js (hunk 2 above)
Comment (Sunbus) [reply views/js/qtiCreator/plugins/menu/preview.js]: U r right, done

File: views/js/qtiCreator/plugins/menu/preview.js (hunk 3 above)
Comment (Sunbus) [reply views/js/qtiCreator/plugins/menu/preview.js]: done

**Comment by oatymart** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**